### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'this.props.styles.fullScreen')

### ISSUE THREAD

**Issue by obrienck** [boxcast/boxcast-sdk-react-native#6]
After clicking the channel list to begin streaming a video, the following error occurs. 

My component is as follows...

export class LinksScreen extends React.Component {
  state = {
    broadcast: null,
    dockable: true,
  };
  showBroadcast(broadcast) {
    this.setState({ broadcast });
  }

  closeBroadcast() {
    this.setState({ broadcast: null });
  }

  render() {
    return (
      <View style={{ flex: 1 }}>
        <View style={{ height: 150, width: "100%" }}>
          <ChannelList
            channelId={"[Removed]"}
            query={"timeframe:relevant timeframe:next"}
            sort={"-starts_at"}
            pageSize={10}
            onSelectBroadcast={(broadcast) => this.setState({ broadcast })}
            horizontal={true}
          />
        </View>
        {this.state.broadcast && (
          <BroadcastModalView
            broadcast={this.state.broadcast}
            dockable={true}
            onDismiss={() => this.setState({ broadcast: null })}
          />
        )}
      </View>
    );
  }
}

The error is as follows...

TypeError: undefined is not an object (evaluating 'this.props.styles.fullScreen')

This error is located at:
    in BroadcastModalView (at LinksScreen.js:69)
    in RCTView (at LinksScreen.js:47)
    in LinksScreen (at SceneView.tsx:98)
    in StaticContainer
    in StaticContainer (at SceneView.tsx:89)
    in EnsureSingleNavigator (at SceneView.tsx:88)
    in SceneView (at useDescriptors.tsx:125)
    in RCTView (at BottomTabView.tsx:42)
    in SceneContent (at BottomTabView.tsx:127)
    in RCTView (at ResourceSavingScene.tsx:44)
    in RCTView (at ResourceSavingScene.tsx:27)
    in ResourceSavingScene (at BottomTabView.tsx:122)
    in RCTView (at screens.native.js:132)
    in ScreenContainer (at BottomTabView.tsx:106)
    in RCTView (at BottomTabView.tsx:105)
    in SafeAreaProviderCompat (at BottomTabView.tsx:104)
    in BottomTabView (at createBottomTabNavigator.tsx:41)
    in BottomTabNavigator (at BottomTabNavigator.js:18)
    in BottomTabNavigator (at SceneView.tsx:98)
    in StaticContainer
    in StaticContainer (at SceneView.tsx:89)
    in EnsureSingleNavigator (at SceneView.tsx:88)
    in SceneView (at useDescriptors.tsx:125)
    in RCTView (at CardContainer.tsx:199)
    in RCTView (at CardContainer.tsx:198)
    in RCTView (at CardSheet.tsx:33)
    in ForwardRef(CardSheet) (at Card.tsx:526)
    in RCTView (at createAnimatedComponent.js:151)
    in AnimatedComponent (at Card.tsx:508)
    in PanGestureHandler (at GestureHandler.native.tsx:13)
    in PanGestureHandler (at Card.tsx:502)
    in RCTView (at createAnimatedComponent.js:151)
    in AnimatedComponent (at Card.tsx:498)
    in RCTView (at Card.tsx:492)
    in Card (at CardContainer.tsx:164)
    in CardContainer (at CardStack.tsx:497)
    in RCTView (at Screens.tsx:70)
    in MaybeScreen (at CardStack.tsx:490)
    in RCTView (at Screens.tsx:48)
    in MaybeScreenContainer (at CardStack.tsx:388)
    in CardStack (at StackView.tsx:433)
    in KeyboardManager (at StackView.tsx:431)
    in RNCSafeAreaView (at src/index.tsx:28)
    in SafeAreaProvider (at SafeAreaProviderCompat.tsx:42)
    in SafeAreaProviderCompat (at StackView.tsx:428)
    in RCTView (at StackView.tsx:427)
    in StackView (at createStackNavigator.tsx:82)
    in StackNavigator (at App.js:22)
    in EnsureSingleNavigator (at BaseNavigationContainer.tsx:288)
    in ForwardRef(BaseNavigationContainer) (at NavigationContainer.tsx:64)
    in ThemeProvider (at NavigationContainer.tsx:63)
    in ForwardRef(NavigationContainer) (at App.js:21)
    in RCTView (at App.js:19)
    in App (at withExpoRoot.js:26)
    in ExpoRoot (at renderApplication.js:40)
    in RCTView (at AppContainer.js:101)
    in DevAppContainer (at AppContainer.js:115)
    in RCTView (at AppContainer.js:119)
    in AppContainer (at renderApplication.js:39)

renderDefaultView
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:169505:35
render
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:169577:98
finishClassComponent
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:18062:43
updateClassComponent
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:18025:50
invokeGuardedCallbackImpl
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:9302:21
invokeGuardedCallback
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:9398:42
beginWork$$1
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:23519:34
performUnitOfWork
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:22657:30
workLoopSync
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:22639:45
renderRoot
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:22403:29
renderRoot
    [native code]:0
runRootCallback
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:22164:34
runRootCallback
    [native code]:0
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:12894:38
unstable_runWithPriority
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:42781:30
flushSyncCallbackQueueImpl
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:12889:28
flushSyncCallbackQueue
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:12878:35
batchedUpdates$1
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:22231:35
batchedUpdates
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:10720:36
_receiveRootNodeIDEvent
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:10771:23
receiveTouches
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:10801:34
__callFunction
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:3087:49
<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:2800:31
__guard
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:3041:15
callFunctionReturnFlushedQueue
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:2799:21
callFunctionReturnFlushedQueue
    [native code]:0

**Comment by bikegriffith** [comment]
@obrienck What version of React Native and React Native Video are you using?

**Comment by obrienck** [comment]
It has taken me awhile to reply because I was attempting to create a React Native app using Expo. This is not possible without ejecting because of the react-native-video dependency. I assumed that was my issue.

So, I have now created a React Native project without Expo and still have the same issue. The fix is very simple, I just grabbed the styles from https://github.com/boxcast/boxcast-sdk-react-native/blob/master/src/BroadcastModalView.js and passed that to the BroadcastModalView and it fixed my issue. The styles exist locally, but the component appears to be referencing the prop instead of the local instance.
